fetch.bs

```diff
@@ -8576,6 +9047,135 @@ with a <var>promise</var>, <var>request</var>, <var>responseObject</var>, and an
 </ol>
 </div>
 
+<p>A {{FetchLaterResult}} has an associated <dfn for=FetchLaterResult>activated getter steps</dfn>,
+which is an algorithm returning a boolean.
+
+<div algorithm>
+<p>The <dfn attribute for=FetchLaterResult><code>activated</code></dfn> getter steps are to call
```

Comment:
```suggestion
<p>The <dfn attribute for=FetchLaterResult><code>activated</code></dfn> getter steps are to return the result of running
```
and wrapping

Reply:
This also can't be resolved by just committing the suggestion as the wrapping will be wrong.

Reply:
Yea, applying the suggestions and fixing the wrapping separately.

Reply:
Should be good now.